bakery/src/tasks/fetch-book.js

```diff
@@ -30,6 +31,7 @@ const task = (taskArgs) => {
           cd fetched-book
           book_dir="$(cat ../book/collection_id)"
           yes | neb get -r -d "$book_dir/raw" "$(cat ../book/server)" "$(cat ../book/collection_id)" "$(cat ../book/version)"
+          wget ${bookSlugsUrl} -O "$book_dir/book-slugs.json"
```

Comment:
Yes, it will. I think we'll end up needing / using it in the preview part of COPS when we implement [cnx 1163](https://github.com/openstax/cnx/issues/1163).